new-frontiers-14/frontier-station-14
# Fix Bluespace Events

## About the PR



Fixes bluespace events, puts them on a separate timer from other in-game events, to ensure events are decently spaced apart.

Removes rewards for most grid values for bluespace events, sets the cache and vault events' rewards to 1 (from 3.3 and 0.7).

Moves a lot of maxOccurrences to reoccurrenceDelay (more suitable for longer shifts, e.g. on Hypatia).

## Why / Balance


I goofed.  The ability to have multiple schedulers should result in more predictable, less chaotic rounds, or at least, more stability/flexibility in scheduling events if changes are needed.

## How to test


1. Set the minimum/maximum time on the BluespaceEventScheduler to very small values, remove earliestStart and minimumPlayers.
2. Spawn, wait about 2 minutes.
3. Attention.  Attention.  Boo-weeeeep.  All events should run.

## Media


## Requirements


## Breaking changes


**Changelog**



:cl:
- fix: Bluespace events should now spawn again, and are scheduled separately from other events.
- tweak: False alarms are more rare, NFSD/Frontier bank accounts are rewarded less for event completion.
- remove: The CentDrobe has been removed from the Vault bluespace event.
